packages/settings-library/src/settings_library/r_clone.py

```diff
@@ -0,0 +1,17 @@
+from enum import Enum
+
+from pydantic import Field
+from .base import BaseCustomSettings
+from .s3 import S3Settings
+
+
+class S3Provider(str, Enum):
+    AWS = "AWS"
+    CEPH = "CEPH"
+    MINIO = "MINIO"
+
+
+class RCloneSettings(BaseCustomSettings):
+    R_CLONE_S3: S3Settings = Field(auto_default_from_env=True)
+    R_CLONE_PROVIDER: S3Provider
+    R_CLONE_REGION: str = Field("us-east-1", description="S3 region to use")
```

Comment:
Q: why not move this to S3Settings?

Reply:
While I wanted to put it inside the `S3Settings`, it is something only used by r-clone since mainly it is a default. That's why it ended up on the `RCloneSettings`. Somehow it did not seem right to have it on the `S3Settings` object.
What would you do? I am open to changing it, since neither of the options convince me.

Reply:
while I agree it's currently only used by RClone, it is still a S3 setting. and it will be used in AWS deployments cause currently we probably use some default. Therefore I would put it there. Can you please check how storage connects to AWS S3 then?

Reply:
Storage uses `Minio` as seen below. It does not use the region, which by default None as a parameter. Not specifying the region in s3 clients equals to using `us-east-1` 

https://github.com/ITISFoundation/osparc-simcore/blob/122703f6cbd964701143d085f3b7a4203e33ec15/services/storage/src/simcore_service_storage/s3wrapper/s3_client.py#L21-L37